_protected/app/system/modules/payment/views/base/tpl/main/pay.tpl

```diff
@@ -16,7 +16,7 @@
     {if !$is_paypal AND !$is_stripe AND !$is_braintree AND !$is_2co AND !$is_skeerel AND !$is_ccbill}
         <p class="err_msg">{lang 'No Payment System Enabled!'}</p>
     {else}
-        {if $membership->enable == 1 AND $membership->price != 0}
+        {if $membership->enable == 1 AND $membership->price >= 0}
```

Comment:
Why would you have a negative price for your paid membership..? I'm not sure to understand that one

Reply:
@pH-7 sorry! I was looking at this in the wrong way and closed this PR once I undersood it

Reply:
No worries